spec/test_app/.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.7.3
```

Comment:
Can you symlink this file to the top-level `.ruby-version`?  They're not really independent.

Reply:
Since we were "forced" to set this gem to 2.6.6, I thought it'd be nice to show that it works for apps that use newer versions of ruby too.

Reply:
I think that's covered by the `requird_ruby_version`: https://github.com/alphagov/govuk_personalisation/blob/98a3cff806cb37a9bf957fabe146b9d3a90a8b1e/govuk_personalisation.gemspec#L17

If we knew this gem didn't work with ruby 2.7, we should add a constraint there.  Since there isn't such a constraint, it can be assumed to work.

Reply:
How about I just delete this file? We don't even need it.

Reply:
:+1: